src/Core/Content/ProductExport/SalesChannel/ExportController.php

```diff
@@ -80,8 +80,13 @@ public function index(Request $request): Response
         $contentType = $this->getContentType($productExport->getFileFormat());
         $encoding = $productExport->getEncoding();
 
-        return (new Response($content ?: null, 200, ['Content-Type' => $contentType . ';charset=' . $encoding]))
-            ->setCharset($encoding);
+        $response = new Response(
+            $content ?: null, 200, [
```

Comment:
dat code style :grimacing: 

![](https://media2.giphy.com/media/3o6Zta3GHWlBxtAW7m/giphy.gif)

Reply:
It was a preset in PHPStorm! I changed it manually to have one argument per line but the formatting broke it again :-/